Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1259,6 +1259,14 @@ working, take a look at `this KEP
     free mode, make sure that default Kubernetes services like ``kube-dns`` and ``kubernetes``
     have the required label value.
 
+Topology Aware Hints
+********************
+
+The kube-proxy replacement implements the K8s service
+`Topology Aware Hints <https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints>`__.
+This allows Cilium nodes to prefer service endpoints residing in the same zone.
+To enable the feature, set ``loadBalancer.serviceTopology=true``.
+
```

Comment:
We could later add a small usage example, otoh referring to the doc might be okay as well. Small hands-on example is quite nice though given user can take this and play with it in their env. Does kubeadm need anything to enable it, we could add it in there as well given our main KPR guide refers to kubeadm?

Reply:
Yeah, it's still alpha feature, so it needs to be enabled via the feature gate. Luckily, the linked page explains how to do this, so I wanted to avoid repeating this info in our guide.

Reply:
It'll be good to call out these caveats from the k8s documentation - 

`Impossible to achieve balanced allocation: In some cases, it will be impossible to achieve a balanced allocation of endpoints among zones. For example, if zone-a is twice as large as zone-b, but there are only 2 endpoints, an endpoint allocated to zone-a may receive twice as much traffic as zone-b. The controller does not assign hints if it can't get this "expected overload" value below an acceptable threshold for each zone. Importantly this is not based on real-time feedback. It is still possible for individual endpoints to become overloaded.`

Reply:
To keep the guide's size sane and also considering that those details will change, I'd prefer to keep only the link to the K8s page with all heuristics.